Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM node:18
+
+# Create app directory
+WORKDIR /usr/src/app
```

Comment:
try to cache package.json file so we dont need to install dependencies every time